### PR TITLE
Fix ModelManager's Filtering to AND-Filter.

### PR DESCRIPTION
**Pull request by ChuaPingChan**

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -71,18 +73,19 @@ public synchronized void addEntry(ReadOnlyEntry entry) {
...
+    public void updateEntry(ReadOnlyEntry target, ReadOnlyEntry editedEntry) throws EntryNotFoundException {
         requireAllNonNull(target, editedEntry);
 
         entryBook.updateEntry(target, editedEntry);
         indicateEntryBookChanged();
     }
 
-    //=========== Filtered Entry List Accessors =============================================================
+    // =========== Filtered Entry List Accessors
+    // =============================================================
```
Comment (kevinLamKB) [inline src/main/java/seedu/address/model/ModelManager.java]: maybe just remove some of the ==== on the right side, lol

Comment (ChuaPingChan) [reply src/main/java/seedu/address/model/ModelManager.java]: @kevinLamKB 
Okay, lol Eclipse auto-format messed up the comments quite a bit. The only change I did is at lines 103-106.

**Comment by ChuaPingChan** [comment]
Fix #39.

**Review by mattheuslee — APPROVED** [review]

**Comment by ChuaPingChan** [comment]
Sorry it's not working well yet. #39

**Pull request closed without merging**